api/symphony-api.json

```diff
@@ -13,17 +13,23 @@
       "lng": "-122.12826"
     },
     "currentSite": {
-      "baseUrl": "http://symphony-service:8080/v1alpha2/",
+      "baseUrl": "https://symphony-service:8081/v1alpha2/",
```

Comment:
I don't think we can run standalone mode with https. We didn't bind 8081 port and we don't have a cert generator.